RecoVertex/Configuration/python/RecoVertex_phase2_timing_cff.py

```diff
@@ -0,0 +1,75 @@
+                                                                                    ptMax=9e99,
+                                                                                    ptErrorCut=9e99)
+trackRefsForJetsBeforeSorting4Dfastsim = trackRefsForJets.clone(src="trackWithVertexRefSelectorBeforeSorting4Dfastsim")
+offlinePrimaryVertices4Dfastsim=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4Dfastsim",
+                                                            particles="trackRefsForJetsBeforeSorting4Dfastsim",
+                                                            trackTimeTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModel"),
+                                                            trackTimeResoTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModelResolution"),
+                                                            assignment=dict(useTiming=True))
+offlinePrimaryVertices4DfastsimWithBS=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4Dfastsim:WithBS",
+                                                                  particles="trackRefsForJetsBeforeSorting4Dfastsim",
+                                                                  trackTimeTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModel"),
+                                                                  trackTimeResoTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModelResolution"),
+                                                                  assignment=dict(useTiming=True))
+
+
+
+unsortedOfflinePrimaryVertices3D = unsortedOfflinePrimaryVertices.clone()
+trackWithVertexRefSelectorBeforeSorting3D = trackWithVertexRefSelector.clone(vertexTag="unsortedOfflinePrimaryVertices3D",
+                                                                             ptMax=9e99,
+                                                                             ptErrorCut=9e99)
+trackRefsForJetsBeforeSorting3D = trackRefsForJets.clone(src="trackWithVertexRefSelectorBeforeSorting3D")
+offlinePrimaryVertices3D = sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices3D",
+                                                       particles="trackRefsForJetsBeforeSorting3D")
+offlinePrimaryVertices3DWithBS = offlinePrimaryVerticesWithBS.clone(vertices="unsortedOfflinePrimaryVertices3D:WithBS",
+                                                                    particles="trackRefsForJetsBeforeSorting3D")
+
+from SimTracker.TrackerHitAssociation.tpClusterProducer_cfi import tpClusterProducer
+from SimTracker.TrackAssociatorProducers.quickTrackAssociatorByHits_cfi import quickTrackAssociatorByHits
+from SimTracker.TrackAssociation.trackTimeValueMapProducer_cfi import trackTimeValueMapProducer
+from CommonTools.RecoAlgos.TOFPIDProducer_cfi import tofPID
```

Comment:
the config parse errors were about this line

Reply:
Yes I had missed this when renaming the file (and didn't notice because of an old .pyc file hanging around)